### DISCUSSION TITLE
Play Gifs on hover in the file gallery

### DISCUSSION THREAD

**Discussion by maximilianspitzer** [diced/zipline#518] [Ideas]
There is intense lag when looking through pages where there are only gifs. Maybe play the gif only when hovering over it with the mouse like with youtube?

**Comment by diced** [comment]
As far as I know this isn't possible without saving an image of the first frame, which I don't want to do for just a gif, I'll try looking into it though and seeing if there are any other ways to go around it since this seems like a good idea to implement in the future.
